src/Management/test/Endpoint.Test/Services/EndpointMiddlewareTest.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Net;
+using System.Text;
+using System.Text.Json;
+using FluentAssertions;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Steeltoe.Common.TestResources;
+using Steeltoe.Logging.DynamicLogger;
+using Steeltoe.Management.Endpoint.Options;
+using Steeltoe.Management.Endpoint.Services;
+using Steeltoe.Management.Endpoint.Test.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Steeltoe.Management.Endpoint.Test.Services;
+
+public class EndpointMiddlewareTest : BaseTest
+{
+    private static readonly Dictionary<string, string?> AppSettings = new()
+    {
+        ["Logging:Console:IncludeScopes"] = "false",
+        ["Logging:LogLevel:Default"] = "Warning",
+        ["Logging:LogLevel:Pivotal"] = "Information",
+        ["Logging:LogLevel:Steeltoe"] = "Information",
+        ["management:endpoints:enabled"] = "true",
+        ["management:endpoints:actuator:exposure:include:0"] = "beans"
+    };
+
+    private readonly ITestOutputHelper _output;
+
+    public EndpointMiddlewareTest(ITestOutputHelper output)
+    {
+        _output = output;
+    }
+
+    [Fact]
+    public async Task HandleServicesRequestAsync_ReturnsExpected()
+    {
+        IOptionsMonitor<ServicesEndpointOptions> opts = GetOptionsMonitorFromSettings<ServicesEndpointOptions>();
+        IOptionsMonitor<ManagementOptions> managementOptions = GetOptionsMonitorFromSettings<ManagementOptions>(AppSettings);
+
+        var configurationBuilder = new ConfigurationBuilder();
+        configurationBuilder.AddInMemoryCollection(AppSettings).Build();
+
+        IServiceCollection serviceCollection = new ServiceCollection();
+
+        // Add some known services 
+        serviceCollection.AddSingleton<ServicesEndpointHandler>();
+        serviceCollection.AddTransient<ServicesEndpointOptions>();
+        serviceCollection.AddScoped<Startup>();
+
+        var ep = new ServicesEndpointHandler(opts, serviceCollection, NullLoggerFactory.Instance);
+        var middle = new ServicesEndpointMiddleware(ep, managementOptions, NullLoggerFactory.Instance);
```

Comment:
During the Management API review, I corrected variable names in tests to not be abbreviations. Can you sync up? For example, see the diff at https://github.com/SteeltoeOSS/Steeltoe/pull/1155/files#diff-9463779c39736a4e999a948765d3cb19cf84e281f5e4b62d406258ab1939f406.